src/components/NavBarComponent.js

```diff
@@ -37,8 +38,16 @@ export default class NavBarComponent extends React.Component {
                                onChange={e => this.setState({searchCity: e.target.value})}/>
                         <a class="btn btn-outline-success my-2 mr-2 my-sm-0"
                            href={`/search/${normalizeCity(this.state.searchCity)}/${this.state.searchQuery}`}>Search</a>
+
+                        {Object.keys(this.props.profile).length == 0 &&
                         <a class="btn btn-outline-success my-2 my-sm-0"
                             href={`/login`}>Log In</a>
+                        }
+                        {Object.keys(this.props.profile).length > 0 &&
+                        <a class="btn btn-outline-danger my-2 my-sm-0 text-danger"
+                            onClick={() => this.props.logout()}
+                            href={`/`}>Log Out</a>
+                        }
```

Comment:
i'm going to bring this up in our call later tonight, but it looks like the the logged-in or logged-out status doesn't update immediately upon initially logging in. might just be something on my end.

Reply:
i just came across a similar issue where if i click `Log Out`, the login/logout button does not change immediately unless i refresh

Reply:
Resolved the Log In issue.  I can't duplicate the log out issue, but lets keep in mind that this may exist and be on the look out for it in the future.